python/do_username/generate.py

```diff
@@ -0,0 +1,34 @@
+import math
+import random
+
+sea_creatures = "walrus seal fish shark clam coral whale crab lobster starfish eel dolphin squid jellyfish ray shrimp mantaRay angler snorkler scubaDiver urchin anemone morel axolotl".split(" ")
+sea_objects = "boat ship submarine yacht dinghy raft kelp seaweed anchor".split(" ")
```

Comment:
```suggestion
SEA_OBJECTS = tuple("boat ship submarine yacht dinghy raft kelp seaweed anchor".split(" "))
```
Uppercase for consistency, this is a constant. Cast to tuple so that it's immutable.